includes/sanitizers/class-amp-base-sanitizer.php

```diff
@@ -661,4 +699,53 @@ public function maybe_add_amp_image_lightbox_node() {
 		);
 		$body_node->appendChild( $amp_image_lightbox );
 	}
+
+	/**
+	 * Parse a style string into an associative array of style attributes.
+	 *
+	 * @param string $style_string Style string to parse.
+	 * @return string[] Associative array of style attributes.
+	 */
+	protected function parse_style_string( $style_string ) {
+		// We need to turn the style string into an associative array of styles first.
+		$style_string = trim( $style_string, " \t\n\r\0\x0B;" );
+		$elements     = preg_split( '/(\s*:\s*|\s*;\s*)/', $style_string );
+
+		if ( 0 !== count( $elements ) % 2 ) {
+			// Style string was malformed, try to process as good as possible by stripping the last element.
+			array_pop( $elements );
+		}
+
+		$chunks = array_chunk( $elements, 2 );
+
+		// phpcs:ignore PHPCompatibility.FunctionUse.NewFunctions.array_columnFound -- WP Core provides a polyfill.
+		return array_combine( array_column( $chunks, 0 ), array_column( $chunks, 1 ) );
```

Comment:
The polyfill is apparently not being loaded because this is causing a test error in PHP 5.4

Reply:
Ha, it was WP-CLI that has the polyfill, not WordPress Core. We have 3 options here:
1.) Rewrite the algorithm to not require `array_column()`.
2.) Pull in `ramsey/array_column` as a dependency.
3.) Copy the code in https://github.com/ramsey/array_column/blob/master/src/array_column.php into our helper functions file.

@westonruter Thoughts?

Reply:
Since we plan to upgrade to PHP 5.6 soon, feel free to do whatever is fastest as a short term solution. I suppose that means copying the code into the helper functions file.

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3209/commits/8794360b3562cad6b6e6887311320b4edc7c08f0